README.md

```diff
@@ -131,7 +131,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [PepsiCo](https://www.pepsicojobs.com/internshipsprograms/jobs/241240?lang=en-us&previousLocale=en-US) | Purchase, NY <br> Plano, TX | Software Development & Engineering Intern (No Sponsorship) |
 | [U.S. Bank](https://careers.usbank.com/global/en/job/2023-0020491/2024-Software-Engineering-Summer-Intern-Early-Careers) | Richfield, MN; and Atlanta, GA | 2024 Software Engineering Summer Intern |
 | Jane Street | New York, NY | [Software Engineer Internship](https://www.janestreet.com/join-jane-street/position/6834728002/) <br/> [Quantitative Trader Internship](https://www.janestreet.com/join-jane-street/position/6850721002/) <br/> [Quantitative Researcher Internship](https://www.janestreet.com/join-jane-street/position/6838982002/) |
-| [General Motors](https://generalmotors.wd5.myworkdayjobs.com/en-US/Careers_GM/job/Detroit-Michigan-United-States-of-America/XMLNAME-2024-Summer-Intern---Digital-Transformation---Data-Analyst-or-Data-Scientist_JR-202313389) | Detroit, MI (Hybrid) | 2024 Summer Intern – Digital Transformation – Data Analyst or Data Scientist (No Sponsorship) |
+| General Motors | Detroit, MI (Hybrid) | [2024 Summer Intern – Digital Transformation – Data Analyst or Data Scientist](https://generalmotors.wd5.myworkdayjobs.com/en-US/Careers_GM/job/Detroit-Michigan-United-States-of-America/XMLNAME-2024-Summer-Intern---Digital-Transformation---Data-Analyst-or-Data-Scientist_JR-202313389) (No Sponsorship) <br/> [2024 Software Engineer Intern](https://search-careers.gm.com/en/jobs/jr-202314588/2024-summer-intern-software-engineering/?utm_campaign=google_jobs_apply&utm_source=google_jobs_apply&utm_medium=organic) (No Sponsorship) |
```

Comment:
```suggestion
| General Motors | Detroit, MI (Hybrid) | [2024 Summer Intern – Digital Transformation – Data Analyst or Data Scientist](https://generalmotors.wd5.myworkdayjobs.com/en-US/Careers_GM/job/Detroit-Michigan-United-States-of-America/XMLNAME-2024-Summer-Intern---Digital-Transformation---Data-Analyst-or-Data-Scientist_JR-202313389) (No Sponsorship) <br/> [2024 Software Engineer Intern](https://generalmotors.wd5.myworkdayjobs.com/en-US/Careers_GM/job/XMLNAME-2024-Summer-Intern---Software-Engineering_JR-202314588) (No Sponsorship) |
```
Let's use this MyWorkdayJobs link instead, because your link actually redirects to GM's MyWorkdayJobs site